src/api.raml

```diff
@@ -497,6 +497,13 @@ traits:
         is:
           - paged: { resources: Collections }
           - subjected: { resources: Collections }
+        queryParameters:
+            containing:
+                description: |
+                    Only return collections containing at least one of these items.
+                type: string[]
+                required: false
+                example: [article/1234, interview/5678]
```

Comment:
Recommendations and metrics use these values, but as part of the path, not of the query string. `:` is also unambiguous. In builder we use `--` (which has its own problems like being recognized as ~strikethrough~ by Jira) but only because other characters are not allowed.